lib/linguist/heuristics.yml

```diff
@@ -371,6 +371,8 @@ disambiguations:
     pattern:
     - '(?i:^\s*\{\$(?:mode|ifdef|undef|define)[ ]+[a-z0-9_]+\})'
     - '^\s*end[.;]\s*$'
+  - language: BitBake
+    pattern: '^(include|require|inherit)\s+[a-z$]'
```

Comment:
This is also waaaay too generic and definitely mis-identifies files from other languages. This is evident from [this search](https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.inc+%2F%5E%28include%7Crequire%7Cinherit%29%5Cs%2B%5Ba-z%24%5D%2F) which shows many Makefiles. As with `.conf`, this is a very generic extension so we need to be incredibly precise with the regex. If in doubt, don't include it.

Reply:
Ok I restricted the regex to the `inherit` part only, which is likely to be present in BitBake `.inc` recipes. It will not match BitBake `.inc` files used in the configuration part but as we removed `.conf` files it's no big deal now.

`inherit` is not a Makefile keyword, this new regex seems to [only match BitBake files](https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.inc+%2F%5Einherit%28%5Cs%2B%5B%5Cw.-%5D%2B%29%2B%5Cs*%24%2F).